sdks/python/setup.py

```diff
@@ -85,7 +85,6 @@ def get_version():
         'apache_beam/utils/counters.py',
         'apache_beam/utils/windowed_value.py',
     ]),
-    setup_requires=['nose>=1.0'],
```

Comment:
We need to find an alternative solution. Because, people getting just the python-sdk part and not wanting to use maven would need this so that `nose` would be installed when they run `python setup.py test`

Would be possible to tell RAT to ignore certain folders.